docker/package/scripts/tezos-accuser-start

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+set -euo pipefail
+
+# $PROTOCOL should be defined in the system unit environment
+accuser="/usr/bin/tezos-accuser-$PROTOCOL"
+
+accuser_dir="$DATA_DIR"
+
+configure_accuser() {
+    local accuser_config="$accuser_dir/config"
+    mkdir -p "$accuser_dir"
+
+    if [ ! -f "$accuser_config" ]; then
+        "$accuser" --base-dir "$accuser_dir" \
+                  --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+                  config init --output "$accuser_config" >/dev/null 2>&1
+    else
+        "$accuser" --base-dir "$accuser_dir" \
+                  --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+                  config update >/dev/null 2>&1
+    fi
+
+}
+
+launch_accuser() {
+    configure_accuser
+    exec "$accuser" --base-dir "$accuser_dir" \
+         --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+         run "$@"
+}
+
+
+launch_accuser "$@"
```

Comment:
Not much of a point in defining functions just to call them once in the same scope.